packages/react/src/side-nav/docs/overview.mdx

```diff
@@ -81,9 +81,118 @@ The background of the `SideNav` must match the background it is being placed on.
 <DoHeading />
 
 - include a concise label
-- indent each nesting level
+- indent each sub-level
 
 <DontHeading />
 
-- create more than 3 levels of nesting
-- place on the right-hand side of content.
+- create more than 3 sub-levels
+- place on the right-hand side of content
+- keep all lists with sub-levels open when there are many items.
+
+## Sub-level items
+
+To help users comprehend the side navigation, choose a strategy that is appropriate for the number of items in the list.
+
+### Visible when active
+
+When the list of items is long, prefer showing the sub-level lists when the item is active. The item is considered active after it is clicked, as the user will be navigated to this page. This is the default behaviour so either setting `subLevelVisible="whenActive"` or omitting this prop will set this strategy.
```

Comment:
Perhaps something like this?
```suggestion
When the list of items is long, prefer showing any sub-level lists when either navigating to the parent item or one of the sub-level items. Since this is the default behaviour, either setting `subLevelVisible="whenActive"` or omitting this prop will use this strategy.
```